_events/2411MasqueradeRevival.md

```diff
@@ -0,0 +1,159 @@
+| {% chara_link Yohack|h1 %}  | 30 |
+| {% chara_link Nessen|h1 %} | 20 |
+| {% chara_link Pubraseer|h2 %} | 20 |
+| {% chara_link Gaisei|h1 %} | 20 |
+| {% chara_link Orbiter|h1 %} | 10 |
+
+
+| Sidekick | +Bonus (%) |
+|-------------|---------------|
+| {% chara_link Yohack %} | 30 |
+| {% chara_link Astosis %}  | 20 | 
+| {% chara_link Nessen %}  | 10 | 
+| {% chara_link Yotaro %}  | 10 | 
+| {% chara_link Orbiter %}  | 10 | 
+| {% chara_link Pubraseer %}  | 10 |
+| {% chara_link Esperdica %}  | 10 | 
+| {% chara_link Okitaka %}  | 10 | 
+| {% chara_link Seiichiro %}  | 10 | 
+| {% chara_link Tsuneaki %}  | 10 | 
+| {% chara_link Raizi %}  | 10 | 
+| {% chara_link Maculata %}  | 10 | 
+| {% chara_link Player %} | 10 | 
+
+## Event Sales Bonus
+
+Event sales bonus added according to how much event hero you have in a single sales, Which is
+{% chara_link Nessen|h2 %}, {% chara_link Astosis|h1 %}, and {% chara_link Yohack|h1 %}
+
+Here is the bonus:
+
+| Hero   | +Bonus(%) |
+|--------|-----------|
+| 1 Event Hero   |     10    |
+| 2 Event Heroes |     30    |
+| 3 Event Heroes |     50    |
+
+## Limited Time Mission
+
+Here is a list of this event's limited time Missions:
+
+| No  | Description      | Rewards      |
+|----|-----------------------------------------------------------|----------------|
+| 1  | Finish any rank event quest 5 times | 500 {% include item-icon.html id=190 %}    |
+| 2  | Finish any rank event quest 10 times | 1 {% include item-icon.html id=34 %}    |
+| 3  | Finish any rank event quest 15 times | 500 {% include item-icon.html id=190 %} |
+| 4  | Finish any rank event quest 20 times | 1 {% include item-icon.html id=34 %}    |
+| 5  | Finish any rank event quest 30 times | 1 {% include item-icon.html id=34 %}    |
+| 6  | Finish any rank event quest 35 times | 750 {% include item-icon.html id=190 %}    |
+| 7  | Finish any rank event quest 40 times | 1 {% include item-icon.html id=34 %}    |
+| 8  | Finish any rank event quest 50 times | 1 {% include item-icon.html id=34 %}    |
+| 9  | Finish any rank event quest 70 times | 1000 {% include item-icon.html id=190 %}    |
+| 10  | Finish any rank event quest 90 times | 1500 {% include item-icon.html id=190 %}    |
+| 11  | Finish event sales 10 times | 1 {% include item-icon.html id=34 %}    |
+| 12 | Clear event free quest rank D  | 400 {% include item-icon.html id=190 %}    |
+| 13 | Clear event free quest rank C  | 400 {% include item-icon.html id=190 %}    |
+| 14 | Clear event free quest rank B  | 600 {% include item-icon.html id=190 %}    |
+| 15 | Clear event free quest rank A  | 600 {% include item-icon.html id=190 %}    |
+| 16 | Clear event free quest rank S  | 800 {% include item-icon.html id=190 %}    |
+| 17 | Clear event free quest beginner  | 600 {% include item-icon.html id=190 %}    |
+| 18 | Clear event free quest expert  | 800 {% include item-icon.html id=190 %}    |
+
+## Event Shop
+
+{% include shop-table.html id=53 %}
```

Comment:
id=72, you can find that out in https://github.com/liveahero-wiki/liveahero-wiki.github.io/commit/ab75611754de23ea8cacfb43e971e08aa39c2552